lite/src/types.rs

```diff
@@ -10,20 +11,22 @@ where
     pub validators: V,  // height H
 }
 
-/// Need to do something better here :)
 pub type Height = u64;
-pub type Hash = u64; // TODO
-pub type Time = u64; // TODO
-pub type Bytes = u64; // TODO
-pub type ValID = u64; // TODO
+/// Size of the underlying Hash in bytes
+pub const HASH_LENGTH: usize = 32;
+#[derive(Eq, PartialEq)]
+pub struct Hash([u8; HASH_LENGTH]);
```

Comment:
I just realized that how I've ended up in #36 was to not re-define the types here at all but rather re-use the types from the tendermint crate:
https://github.com/interchainio/tendermint-rs/pull/36/files#diff-b1b6c7ff77dbf13838f76b8403715185

That implies to make this (lite) a module instead of a separate crate (see commit message here: https://github.com/interchainio/tendermint-rs/pull/36/commits/d3ce237b13e6289b44a92f807ebec0fa8a5b7d00)